winui/Kanban/Getting-started.md

```diff
@@ -0,0 +1,241 @@
+---
+layout: post
+title: Getting Started with WinUI Kanban control | Syncfusion
+description: Learn here all about getting started with Syncfusion WinUI Kanban (SfKanban) control, its elements, and more.
+platform: winui
+control: SfKanban
+documentation: ug
+---
+# Getting Started with WinUI Kanban (SfKanban)
+
+This section provides an overview of working with WinUI Kanban and a step-by-step guide to configuring and using it effectively in your application.
+
+## Creating an application with WinUI Kanban
+
+1. Create a [WinUI 3 desktop app for C# and .NET 5](https://learn.microsoft.com/en-us/windows/apps/winui/winui3/create-your-first-winui3-app).
+
+2. Add reference to the `Syncfusion.Kanban.WinUI` NuGet.
+
+3. Import the control namespace `Syncfusion.UI.Xaml.Kanban` in XAML or C# code.
+
+4. Initialize the WinUI Kanban control.
+
+{% capture codesnippet1 %}
+{% tabs %}
+
+{% highlight xaml tabtitle="MainWindow.xaml" hl_lines="7 12" %}
+<Window x:Class="GettingStarted.MainWindow"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:local="using:GettingStarted"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:kanban="using:Syncfusion.UI.Xaml.Kanban"
+        mc:Ignorable="d"
+        Title="GettingStarted">
+
+        <Grid>
+            <kanban:SfKanban x:Name="sfKanban"/>
+        </Grid>
+</Window>
+
+{% endhighlight %}
+
+{% highlight c# tabtitle="MainWindow.xaml.cs" hl_lines="1 13 14" %}
+using Syncfusion.UI.Xaml.Kanban;
+
+namespace GettingStarted
+{
+    /// <summary>
+    /// An empty window that can be used on its own or navigated to within a Frame.
+    /// </summary>
+    public sealed partial class MainWindow : Window
+    {
+        public MainWindow()
+        {
+            this.InitializeComponent();
+            SfKanban sfKanban = new SfKanban();
+            this.Content = sfKanban;
+        }
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+{% endcapture %}
+{{ codesnippet1 | OrderList_Indent_Level_1 }}
+
```

Comment:
## Populate WinUI kanban item source

Here are the steps to render kanban card items using the WinUI Kanban control with the respective `KanbanModel` class:

- Create the ViewModel.
- Bind the item source for Kanban.

Reply:
added content